### PR TITLE
Update feature country dropdown

### LINKED ISSUE #98: [PROFILES] Have a working Country Selector
## Description
The country selector should work as expected - when a user selects a country, they are taken to the right profile and if they are looking at a country profile (or its sections), the country selector will be showing the right one.

## Screenshots

screen shot 2018-06-19 at 10 44 04

## Tasks

- [x] Determine the profile being viewed, have that country's flag and name as the button's content.

### PR DESCRIPTION
**Pull request by nerdlet**
## Description

Related to #98 

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

## Screenshots

image

image

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: takwimu/templates/takwimu/_includes/report/country_selector.html [hunk 1]
```diff
@@ -1,13 +1,25 @@
 {% load staticfiles %}
 
+{% block content %}
+<script src="https://code.jquery.com/jquery-3.3.1.min.js"></script>
```
Comment (DavidLemayian) [inline takwimu/templates/takwimu/_includes/report/country_selector.html]: Please remove this. It's already loaded in the footer.

File: takwimu/templates/takwimu/_includes/report/country_selector.html [hunk 2]
```diff
@@ -1,13 +1,25 @@
...
   </div>
-</div>
\ No newline at end of file
+</div>
+<!--Enable dropdown select list-->
+<script>
+	$('.dropdown-menu a').click(function(){
+		$(".btn:first-child").html($(this).html()+' <span class="caret"></span>');
+	});
+</script>
```
Comment (DavidLemayian) [inline takwimu/templates/takwimu/_includes/report/country_selector.html]: You'll want to do this in a `window.onload` function to wait for jQuery to be available.

File: takwimu/templates/takwimu/_includes/report/country_selector.html (hunk 1 above)
Comment (nerdlet) [reply takwimu/templates/takwimu/_includes/report/country_selector.html]: Sure thing

File: takwimu/templates/takwimu/_includes/report/country_selector.html (hunk 2 above)
Comment (nerdlet) [reply takwimu/templates/takwimu/_includes/report/country_selector.html]: @DavidLemayian Made the necessary changes and places the scripts in the base layout template

File: DATA.md [hunk 3]
```diff
@@ -73,7 +73,7 @@ Visual representations of the data produced from the values. Types are according
     b. Is disaggregated e.g male and female
 
 
-Types of visusals that are available:
+Types of visusls that are available:
```
[the remark this answers is not in the document]
Comment (nerdlet) [reply DATA.md]: Yea, that was totally unintentional. Apologies for that

File: takwimu/templates/takwimu/_layouts/_base.html [hunk 4]
```diff
@@ -77,6 +77,15 @@
...
   </script>
 
+  <!--Load the Country dropdown-->
+  <script>
+  $(window).on('load', function(){
+    $('.dropdown-menu a').click(function(){
+      $(".btn:first-child").html($(this).html()+' <span class="caret"></span>');
+    });
+  });
+</script>
```
[the remark this answers is not in the document]
Comment (nerdlet) [reply takwimu/templates/takwimu/_layouts/_base.html]: Done

**Pull request merged**